model/QtiJsonItemCompiler.php

```diff
@@ -253,4 +262,54 @@ private function getItemAssetXmlReplacer(): QtiItemAssetXmlReplacer
     {
         return $this->getServiceLocator()->get(QtiItemAssetXmlReplacer::class);
     }
+
+    private function getEnhancedMetadataProperties(): array
```

Comment:
I would recommend create a new class "EnhancedMetadataPropertiesParser" to be used also on test, delivery and other resources contexts, cause it is a requirement for this task. Maybe in `generis` extension, since this is using RDF tripples as base. 

Wdyt?